README.md

```diff
@@ -1,35 +1,55 @@
+Once installed, the Deephaven icon will show up in the activity bar on the left side of VS Code:
 
 ![VS Code Activity Bar](./docs/assets/dh-activity-bar.gif)
 
-The "SERVERS" tree will show the status of any configured servers. 
+By default, the extension is only configured to connect to a single Community server hosted at `http://localhost:10000`. See more about adding, removing, and updating servers in the [Configuration](#configuration) section below.
+
+## Usage
 
+You can run Deephaven code in two ways.
 
-## Running Code
-To run a script against a running server, simply click the `Run Deephaven File` action at the top of a file supported by the server (`python` or `groovy`).
+### Run code against a Deephaven server
 
-![Connect to Community Server](./docs/assets/dhc-connect-to-server.gif)
+You can execute files as queries against a running Deephaven server with this extension. To do so, your file needs to match the language of the server you are connecting to (Python or Groovy). Atop the file, click the `Run Deephaven File` action.
 
-A new connection will show up in the "CONNECTIONS" tree, and the "PANELS" should show any variables exposed on the connection. To disconnect, hover over the connection item and click the trash icon.
+![Run Deephaven file](./docs/assets/dhc-connect-to-server.gif)
+
+A new connection will appear in the [`CONNECTIONS` panel](#connections) on the left hand side of VS Code when you execute the code. Additionally, the [`PANELS` tree](#panels) below will show any variables exposed in the connection. To disconnect, hover over the connection item and click the trash icon.
 
 ### Markdown Code Blocks
-Python and Groovy codeblocks in Markdown files can be run by clicking the "Run Deephaven Block" action directly above the codeblock.
 
-![Markdown Codeblocks](docs/assets/markdown-codeblocks.png)
+The extension also supports running code blocks in markdown files. Like with [running entire files](#run-code-against-a-deephaven-server), the code block must match the language of the server you are connecting to. Atop the code block, click the `Run Deephaven Block` action.
+
+![Run Deephaven block](docs/assets/markdown-codeblocks.png)
 
 ## Configuration
 
-A single Community server `http://localhost:10000/` is configured by default and doesn't require any additional config. Additional connections can be configured in `VS Code` settings.
+By default, the extension is configured to connect to a single Community server hosted at `http://localhost:10000`.
+
+To add more servers or change the configuration of the default server, head to VS Code settings. This opens a settings UI, where you can search for the extension with `@ext:deephaven.vscode-deephaven`. It shows two items you can configure:
```

Comment:
If we are providing explicit navigation instructions to VS Code settings, we probably need to mention Workspace settings as well which are accessible via the "Workspace" tab in that Settings UI